tests/trainers/test_bigearthnet.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+from typing import Tuple, cast
+
+import pytest
+from _pytest.fixtures import SubRequest
+
+from torchgeo.trainers import BigEarthNetDataModule
+
+
+class TestBigEarthNetDataModule:
+    @pytest.fixture(
+        params=[
+            ("s1", True),
```

Comment:
Let's use `zip(["s1", "s2", "all"], [True, True, False])` here to cut down on the total number of combinations we test.

Reply:
Also add `scope="class"` so we don't have to reinstantiate the dm each time.